btx/interfaces/ielog.py

```diff
@@ -2,6 +2,41 @@
 import numpy as np
 import shutil
 from glob import glob
+import json
+import requests
+from typing import Optional
+
+def elog_report_post(summary_file: str, update_url: Optional[str] = None):
+    """! Post a summary file to the eLog's run report section.
+
+    @param summary_file (str) Path to the summary file to post.
+    @param update_url (str | None) URL to post to. Attempts to retrieve if None.
+    """
+    with open(summary_file, 'r') as f:
+        data: dict = json.load(f) # load for files instead of loads
+        post_list: list = [ { 'key': f'{key}', 'value': f'{data[key]}' }
+                            for key in data ]
+        if not update_url:
+            url = os.environ.get('JID_UPDATE_COUNTERS')
+            if url:
```

Comment:
Do we want to throw a warning if no URL is found?

Reply:
That's a tough one. We could throw an exception, but we have no way of handling it - that is out of our hands since it comes from the ARP. Could log/print that can't update the elog though.

Reply:
I think a logger.warning message might come in handy?